.github/workflows/update_docker_images.yaml

```diff
@@ -50,9 +52,15 @@ jobs:
 
           make update-docker-image FILE_PATH=${{ matrix.yaml_file_path }} QUERY_STRING='${{ matrix.yaml_value_path }}' DEBUG_MODE=$DEBUG_MODE
 
+      - name: Check if PR is already open
```

Comment:
Can you clarify what is the motivation for doing this? We don't want open PR to be updated on changes to the base branch/main? I don't see any harm in it, so I am probably missing something.

Reply:
Clarified in the description of the PR.